src/apps/vpn/controller.cpp

```diff
@@ -55,6 +56,9 @@ constexpr const int CONNECTION_MAX_RETRY = 9;
 constexpr const uint32_t CONFIRMING_TIMOUT_SEC = 10;
 constexpr const uint32_t HANDSHAKE_TIMEOUT_SEC = 15;
 
+constexpr const uint32_t VPNSESSION_PING_TIMER_SEC = 10800; //3 hours
```

Comment:
Nit - do this is math equation
```suggestion
constexpr const uint32_t VPNSESSION_PING_TIMER_SEC = 3 * 60 * 60; //3 hours
```